examples/feeds/create_feed_item_set.rb

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Creates a new feed item set for a specified feed, which must belong to either
+# a Google Ads location extension or an affiliate extension. This is equivalent
+# to a "location group" in the Google Ads UI. See
+# https://support.google.com/google-ads/answer/9288588 for more detail.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def create_feed_item_set(customer_id, feed_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  operation = client.operation.create_resource.feed_item_set do |set|
+    set.feed = client.path.feed(customer_id, feed_id)
+    set.display_name = "Feed Item Set ##{(Time.new.to_f * 1000).to_i}"
+
+    # A feed item set can be created as a dynamic set by setting an optional
+    # filter field below. If your feed is a location extension, uncomment the
+    # code that sets dynamic_location_set_filter. If your feed is an affiliate
+    # extension, set dynamic_affiliate_location_set_filter instead.
+
+		# 1) Location extension.
+    # set.dynamic_location_set_filter = client.resource.dynamic_location_set_filter do |filter|
+    #   filter.business_name_filter = client.resource.business_name_filter do |bnf|
+    #     bnf.business_name = "INSERT_YOUR_BUSINESS_NAME_HERE"
+    #     bnf.filter_type = :EXACT
+    #   end
+		# end
```

Comment:
Some lines have strange indentation. Is this intended?

Reply:
Ugh, there are tabs intermixed with spaces. It looked fine in my IDE. Thanks for catching!